mvpa2/base/externals.py

```diff
@@ -208,8 +211,9 @@ def __check_shogun(bottom_version, custom_versions=[]):
     if (ver in custom_versions) or (ver >= bottom_version):
         return True
     else:
-        raise ImportError, 'Version %s is smaller than needed %s' % \
-              (ver, bottom_version)
+        raise ImportError(
+            'Version %s is smaller than needed %s'
+            % (ver, bottom_version))
```

Comment:
hm, interesting, forgot (happily) about that one.  IMHO this way is more descriptive, thus less bug prone -- you would immediately see what this line is for -- for string arguments, otherwise you need to jump to prev line to see that.  And PyCharm is not warning about this one, so I guess they also consider it good enough to ignore